.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: DKPro JWPL Forum/Mailing list
+    url: https://groups.google.com/g/jwpl
```

Comment:
Is this still used or switch to Discussions on GitHub instead?

Reply:
Well, last mail there was from this year...